akka-remote/src/main/scala/akka/remote/artery/ArterySettings.scala

```diff
@@ -71,6 +71,13 @@ private[akka] final class ArterySettings private (config: Config) {
   val LogSend: Boolean = getBoolean("log-sent-messages")
   val LogAeronCounters: Boolean = config.getBoolean("log-aeron-counters")
 
+  val Transport: Transport = toRootLowerCase(getString("transport")) match {
```

Comment:
I guess `toRootLowerCase` isn't really needed if we only expect USASCII characters here, but it probably also doesn't matter :)